webapp/pages/post/_id/_slug/index.vue

```diff
@@ -197,7 +197,7 @@ export default {
 
     .ds-card-image {
       img {
-        height: 300px;
+        max-height: 710px;
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-03T23:55:34Z" title="Friday, October 4th 2019, 1:55:34 am +02:00">Oct 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -197,7 +197,7 @@ export default {
 
     .ds-card-image {
       img {
-        height: 300px;
+        max-height: 710px;
```


Has this not been changed by @vbelolapotkov for a some reason? Code creep? :laughing:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-04T07:57:18Z" title="Friday, October 4th 2019, 9:57:18 am +02:00">Oct 4, 2019</time>_

---


it was changed by his `PR` yes... but as I pointed out to him, the reason I didn't like the change is cause we would need to change it when we deal with this issue. There is simply no way to enforce a 1:1 aspect ration with a fixed width unless we also limit the height.
I did double check the scroll functionality to check if it were broken in the way he described and it wasn't, but a lot changed on that `PR` before it was merged.
hehehe code creep is when you change code that doesn't have anything to do with the ticket, as I explained above... this is the _only_ change relevant to the issue :stuck_out_tongue:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-04T07:58:28Z" title="Friday, October 4th 2019, 9:58:28 am +02:00">Oct 4, 2019</time>_

---


Though, I think 710px is not actually a `1:1` ratio, since the `Post` page has a width of `910px`, I guess

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-04T08:00:16Z" title="Friday, October 4th 2019, 10:00:16 am +02:00">Oct 4, 2019</time>_

---


As discussed in the issue about image cropping though, if we are inflexible with the width of the `Post`, then enforcing a max `1:1` means that the user would need to scroll to see anything other than the `Post` image